VideoWeb/VideoWeb.Common/Models/Conference.cs

```diff
@@ -36,6 +36,9 @@ public Conference()
         public bool IsScottish { get; set; }
         public string IngestUrl { get; set; }
         public ConferenceMeetingRoom MeetingRoom { get; set; }
+        public DateTime CreatedDateTime { get; set; }
```

Comment:
Whilst you're in this space would you mind removing the ConferenceState enum from the API and use the enum defined in the commons namespace?